tests/yam/agama/auto/url_prepare.pm

```diff
@@ -0,0 +1,19 @@
+## Copyright 2023 SUSE LLC
```

Comment:
this doesn't need a new test module, it can be implemented inside `bootloader_start` we don't need additional method just for this, in fact the dud is resolved in similar way I think. We can add this for follow-up ticket.